op-program/client/l2/engineapi/test/l2_engine_api_tests.go

```diff
@@ -403,6 +403,7 @@ func (h *testHelper) startBlockBuilding(head *types.Header, newBlockTimestamp et
 		Transactions:          txData,
 		NoTxPool:              true,
 		GasLimit:              &gasLimit,
+		EIP1559Params:         new(eth.Bytes8),
```

Comment:
wondering if these should actually be set to the ones from the deploy config, or SystemConfig? These shouldn't be 0 on Holocene chains.

Reply:
This is to make the test pass, otherwise it fails [here](https://github.com/ethereum-optimism/optimism/blob/2b34218fead9c5ea5608ab6be17ae6d53576e814/op-program/client/l2/engineapi/l2_engine_api.go#L284). Looks like there's no easy way to get access to deploy config or SystemConfig at this spot?